### PR TITLE
Jon/merge2

### PR DESCRIPTION
**Pull request by jonborchardt**
another merge

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -48,7 +48,11 @@ Describe changes from userside, and list all potential break changes or other ri
 
 ### ☑️ Self Check before Merge
 
+<<<<<<< HEAD
 ⚠️ Please check all items below before review. ⚠️
+=======
+⚠️ Please check all items below before review. ⚠️ 
+>>>>>>> c19685e0ea1be170025e91b33d1371cd39fa7dd1
```
Comment (codeviking) [inline .github/PULL_REQUEST_TEMPLATE.md:55]: Looks like a merge conflict that may be left over here?

Comment (jonborchardt) [reply .github/PULL_REQUEST_TEMPLATE.md:55]: thanks

File: .github/PULL_REQUEST_TEMPLATE/pr_cn.md [hunk 2]
```diff
@@ -48,7 +48,11 @@
 
 ### ☑️ 请求合并前的自查清单
 
+<<<<<<< HEAD
 ⚠️ 请自检并全部**勾选全部选项**。⚠️
+=======
+⚠️ 请自检并全部**勾选全部选项**。⚠️ 
+>>>>>>> c19685e0ea1be170025e91b33d1371cd39fa7dd1
```
Comment (codeviking) [inline .github/PULL_REQUEST_TEMPLATE/pr_cn.md]: ☝️ Maybe this too?

File: .github/workflows/rebase.yml [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
   rebase:
     name: Rebase
     if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase')
     runs-on: ubuntu-latest
     steps:
+<<<<<<< HEAD
     - uses: actions/checkout@master
+=======
+    - uses: actions/checkout@v2
+>>>>>>> c19685e0ea1be170025e91b33d1371cd39fa7dd1
```
Comment (codeviking) [inline .github/workflows/rebase.yml]: ☝️

File: .github/PULL_REQUEST_TEMPLATE/pr_cn.md (hunk 2 above)
Comment (jonborchardt) [reply .github/PULL_REQUEST_TEMPLATE/pr_cn.md]: yes, i had 'hidden files' with conflicts, i got them all now

File: .gitignore [hunk 4]
```diff
@@ -58,6 +58,8 @@ site/theme/template/Resources/**/*.jsx
 site/theme/template/NotFound.jsx
 scripts/previewEditor/index.html
 
+components/version/version.tsx
```
Comment (codeviking) [inline .gitignore:61]: Do we generate this, or does `antd`?

Comment (jonborchardt) [reply .gitignore:61]: antd

**Pull request merged**